### ISSUE TITLE
error: use of undeclared identifier 'MobileFFprobe'

### ISSUE THREAD

**Issue by NickBusey** [tanersener/react-native-ffmpeg#156]
**Description**

Trying to use ffmpeg, and running `yarn run ios` results in errors.

I know I'm probably doing something wrong, just not sure what. I have run `pod install` several times, seems to complete fine.

**Expected behavior**

No errors.

**Current behavior**

```
/Users/nick/Code/QuikVid/node_modules/react-native-ffmpeg/ios/RNFFmpegModule.m:97:19: error: use of undeclared identifier 'MobileFFprobe'
        int rc = [MobileFFprobe executeWithArguments:arguments];
                  ^
/Users/nick/Code/QuikVid/node_modules/react-native-ffmpeg/ios/RNFFmpegModule.m:183:47: error: use of undeclared identifier 'MobileFFprobe'
        MediaInformation *mediaInformation = [MobileFFprobe getMediaInformation:path];
```

**Logs**

```
yarn run ios
......
    cd /Users/nick/Code/QuikVid/ios/Pods
    export LANG=en_US.US-ASCII
    /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang -x objective-c -target x86_64-apple-ios9.0-simulator -fmessage-length=0 -fdiagnostics-show-note-include-stack -fmacro-backtrace-limit=0 -std=gnu11 -fobjc-arc -fobjc-weak -fmodules -fmodules-cache-path=/Users/nick/Library/Developer/Xcode/DerivedData/ModuleCache.noindex -fmodules-prune-interval=86400 -fmodules-prune-after=345600 -fbuild-session-file=/Users/nick/Library/Developer/Xcode/DerivedData/ModuleCache.noindex/Session.modulevalidation -fmodules-validate-once-per-build-session -Wnon-modular-include-in-framework-module -Werror=non-modular-include-in-framework-module -Wno-trigraphs -fpascal-strings -O0 -fno-common -Wno-missing-field-initializers -Wno-missing-prototypes -Werror=return-type -Wdocumentation -Wunreachable-code -Wno-implicit-atomic-properties -Werror=deprecated-objc-isa-usage -Wno-objc-interface-ivars -Werror=objc-root-class -Wno-arc-repeated-use-of-weak -Wimplicit-retain-self -Wduplicate-method-match -Wno-missing-braces -Wparentheses -Wswitch -Wunused-function -Wno-unused-label -Wno-unused-parameter -Wunused-variable -Wunused-value -Wempty-body -Wuninitialized -Wconditional-uninitialized -Wno-unknown-pragmas -Wno-shadow -Wno-four-char-constants -Wno-conversion -Wconstant-conversion -Wint-conversion -Wbool-conversion -Wenum-conversion -Wno-float-conversion -Wnon-literal-null-conversion -Wobjc-literal-conversion -Wshorten-64-to-32 -Wpointer-sign -Wno-newline-eof -Wno-selector -Wno-strict-selector-match -Wundeclared-selector -Wdeprecated-implementations -DPOD_CONFIGURATION_DEBUG=1 -DDEBUG=1 -DCOCOAPODS=1 -DOBJC_OLD_DISPATCH_PROTOTYPES=0 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.2.sdk -fasm-blocks -fstrict-aliasing -Wprotocol -Wdeprecated-declarations -g -Wno-sign-conversion -Winfinite-recursion -Wcomma -Wblock-capture-autoreleasing -Wstrict-prototypes -Wno-semicolon-before-method-body -Wunguarded-availability -fobjc-abi-version=2 -fobjc-legacy-dispatch -index-store-path /Users/nick/Library/Developer/Xcode/DerivedData/QuikVid-besltckxxdhmcactkggkgxxrleno/Index/DataStore -iquote /Users/nick/Library/Developer/Xcode/DerivedData/QuikVid-besltckxxdhmcactkggkgxxrleno/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/react-native-ffmpeg.build/react-native-ffmpeg-generated-files.hmap -I/Users/nick/Library/Developer/Xcode/DerivedData/QuikVid-besltckxxdhmcactkggkgxxrleno/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/react-native-ffmpeg.build/react-native-ffmpeg-own-target-headers.hmap -I/Users/nick/Library/Developer/Xcode/DerivedData/QuikVid-besltckxxdhmcactkggkgxxrleno/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/react-native-ffmpeg.build/react-native-ffmpeg-all-target-headers.hmap -iquote /Users/nick/Library/Developer/Xcode/DerivedData/QuikVid-besltckxxdhmcactkggkgxxrleno/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/react-native-ffmpeg.build/react-native-ffmpeg-project-headers.hmap -I/Users/nick/Library/Developer/Xcode/DerivedData/QuikVid-besltckxxdhmcactkggkgxxrleno/Build/Products/Debug-iphonesimulator/react-native-ffmpeg/include -I/Users/nick/Code/QuikVid/ios/Pods/Headers/Private -I/Users/nick/Code/QuikVid/ios/Pods/Headers/Private/react-native-ffmpeg -I/Users/nick/Code/QuikVid/ios/Pods/Headers/Public -I/Users/nick/Code/QuikVid/ios/Pods/Headers/Public/DoubleConversion -I/Users/nick/Code/QuikVid/ios/Pods/Headers/Public/React-Core -I/Users/nick/Code/QuikVid/ios/Pods/Headers/Public/React-RCTBlob -I/Users/nick/Code/QuikVid/ios/Pods/Headers/Public/React-RCTText -I/Users/nick/Code/QuikVid/ios/Pods/Headers/Public/React-cxxreact -I/Users/nick/Code/QuikVid/ios/Pods/Headers/Public/React-jsi -I/Users/nick/Code/QuikVid/ios/Pods/Headers/Public/React-jsiexecutor -I/Users/nick/Code/QuikVid/ios/Pods/Headers/Public/React-jsinspector -I/Users/nick/Code/QuikVid/ios/Pods/Headers/Public/Yoga -I/Users/nick/Code/QuikVid/ios/Pods/Headers/Public/glog -I/Users/nick/Code/QuikVid/ios/Pods/Headers/Public/mobile-ffmpeg-https -I/Users/nick/Code/QuikVid/ios/Pods/Headers/Public/react-native-ffmpeg -I/Users/nick/Library/Developer/Xcode/DerivedData/QuikVid-besltckxxdhmcactkggkgxxrleno/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/react-native-ffmpeg.build/DerivedSources-normal/x86_64 -I/Users/nick/Library/Developer/Xcode/DerivedData/QuikVid-besltckxxdhmcactkggkgxxrleno/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/react-native-ffmpeg.build/DerivedSources/x86_64 -I/Users/nick/Library/Developer/Xcode/DerivedData/QuikVid-besltckxxdhmcactkggkgxxrleno/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/react-native-ffmpeg.build/DerivedSources -F/Users/nick/Library/Developer/Xcode/DerivedData/QuikVid-besltckxxdhmcactkggkgxxrleno/Build/Products/Debug-iphonesimulator/react-native-ffmpeg -F/Users/nick/Code/QuikVid/ios/Pods/mobile-ffmpeg-https -include /Users/nick/Code/QuikVid/ios/Pods/Target\ Support\ Files/react-native-ffmpeg/react-native-ffmpeg-prefix.pch -MMD -MT dependencies -MF /Users/nick/Library/Developer/Xcode/DerivedData/QuikVid-besltckxxdhmcactkggkgxxrleno/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/react-native-ffmpeg.build/Objects-normal/x86_64/RNFFmpegModule.d --serialize-diagnostics /Users/nick/Library/Developer/Xcode/DerivedData/QuikVid-besltckxxdhmcactkggkgxxrleno/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/react-native-ffmpeg.build/Objects-normal/x86_64/RNFFmpegModule.dia -c /Users/nick/Code/QuikVid/node_modules/react-native-ffmpeg/ios/RNFFmpegModule.m -o /Users/nick/Library/Developer/Xcode/DerivedData/QuikVid-besltckxxdhmcactkggkgxxrleno/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/react-native-ffmpeg.build/Objects-normal/x86_64/RNFFmpegModule.o
/Users/nick/Code/QuikVid/node_modules/react-native-ffmpeg/ios/RNFFmpegModule.m:97:19: error: use of undeclared identifier 'MobileFFprobe'
        int rc = [MobileFFprobe executeWithArguments:arguments];
                  ^
/Users/nick/Code/QuikVid/node_modules/react-native-ffmpeg/ios/RNFFmpegModule.m:183:47: error: use of undeclared identifier 'MobileFFprobe'
        MediaInformation *mediaInformation = [MobileFFprobe getMediaInformation:path];
                                              ^
2 errors generated.


2020-05-06 13:43:17.343 XCBBuildService[52497:1251936] Failed to remove: /Users/nick/Library/Developer/Xcode/DerivedData/QuikVid-besltckxxdhmcactkggkgxxrleno/Build/Intermediates.noindex/XCBuildData/bc6ff0738df839adf87bedbe57a1c80c-desc.xcbuild: unlink(/Users/nick/Library/Developer/Xcode/DerivedData/QuikVid-besltckxxdhmcactkggkgxxrleno/Build/Intermediates.noindex/XCBuildData/bc6ff0738df839adf87bedbe57a1c80c-desc.xcbuild): No such file or directory (2)
** BUILD FAILED **


The following build commands failed:
        CompileC /Users/nick/Library/Developer/Xcode/DerivedData/QuikVid-besltckxxdhmcactkggkgxxrleno/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/react-native-ffmpeg.build/Objects-normal/x86_64/RNFFmpegModule.o /Users/nick/Code/QuikVid/node_modules/react-native-ffmpeg/ios/RNFFmpegModule.m normal x86_64 objective-c com.apple.compilers.llvm.clang.1_0.compiler
(1 failure)

error Command failed with exit code 1.
```

**Environment**

"react-native": "~0.61.5",

**Comment by NickBusey** [comment]
Add `#import <mobileffmpeg/MobileFFprobe.h>` to `RNFFmpegModule.m` seems to fix this.

**Comment by tanersener** [comment]
I didn't receive a similar complaint before. 
I'm not sure about the reason but importing `MobileFFprobe.h` in the `RNFFmpegModule.m` is not too difficult. I can add it in the next release.

**Comment by tanersener** [comment]
Fixed in `v0.5.0`.
